src/rpc/types.ts

```diff
@@ -11,3 +11,10 @@ export interface RpcClient {
 }
 
 export type RpcWorker = WithIndexedOperations<RpcClient>;
+
+// Re-export registrar utilities for convenience
```

Comment:
why did you re-export this here instead of using them directly from registrar.ts?